tests/js/panelsTests.js

```diff
@@ -382,23 +394,108 @@ https://github.com/gpii/universal/LICENSE.txt
+            "contrast-bw": "black on white",
+            "contrast-wb": "white on black"
+        },
+        themeLabels: ["no change", "black on white", "white on black"]
+    });
+
+    fluid.defaults("gpii.tests.contrastPanel", {
+        gradeNames: ["fluid.test.testEnvironment", "autoInit"],
+        components: {
+            contrast: {
+                type: "gpii.tests.prefs.panel.contrast",
+                container: ".gpiic-fd-contrast"
+            },
+            contrastTester: {
+                type: "gpii.tests.contrastTester"
+            }
+        }
+    });
+
+    fluid.defaults("gpii.tests.contrastTester", {
+        gradeNames: ["fluid.test.testCaseHolder", "autoInit"],
+        modules: [{
+            name: "Test the contrast settings panel",
+            tests: [{
+                expect: 5,
+                name: "Rendering",
+                sequence: [{
+                    func: "{contrast}.refreshView"
+                }, {
+                    listener: "gpii.tests.contrastTester.verifyRendering",
+                    event: "{contrast}.events.afterRender"
+                }]
+            }, {
+                expect: 3,
+                name: "Selection",
+                sequence: [{
+                    func: "gpii.tests.prefs.panel.utils.triggerRadioButton",
+                    args: ["{contrast}.dom.themeInput", 1]
+                }, {
+                    listener: "gpii.tests.contrastTester.verifyModel",
+                    args: ["{contrast}", "bw"],
+                    spec: {path: "value", priority: "last"},
+                    changeEvent: "{contrast}.applier.modelChanged"
+                }, {
+                    func: "gpii.tests.prefs.panel.utils.triggerRadioButton",
+                    args: ["{contrast}.dom.themeInput", 2]
+                }, {
+                    listener: "gpii.tests.contrastTester.verifyModel",
+                    args: ["{contrast}", "wb"],
+                    spec: {path: "value", priority: "last"},
+                    changeEvent: "{contrast}.applier.modelChanged"
+                }, {
+                    func: "gpii.tests.prefs.panel.utils.triggerRadioButton",
+                    args: ["{contrast}.dom.themeInput", 0]
+                }, {
+                    listener: "gpii.tests.contrastTester.verifyModel",
+                    args: ["{contrast}", "default"],
+                    spec: {path: "value", priority: "last"},
+                    changeEvent: "{contrast}.applier.modelChanged"
+                }]
+            }]
+        }]
+    });
+
+    gpii.tests.contrastTester.verifyRendering = function (that) {
+        jqUnit.assertEquals("The instructions should have been set correctly.", that.options.messageBase.instructions, that.locate("instructions").text());
+        gpii.tests.prefs.panel.utils.verifyRadioButtonRendering(that.locate("themeInput"), that.locate("themeLabel"), that.options.themeLabels, that.model.value);
```

Comment:
Should also verify corresponding themes have been applied to buttons, what you do in `that.style()`.